build/launcher.Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN echo \
 
 RUN apt-get update && \
     apt-get install -yq --no-install-recommends \
-        containerlab=0.45.* \
+            containerlab=0.0.0~14789vxlan \
```

Comment:
@carlmontanari I wonder, maybe we should use `ARG` or ENV for this stuff and set it to a default value that it used to be before and then we can use an input for it in the custom release dispatch?